.github/workflows/examples-components.yml

```diff
@@ -168,10 +179,37 @@ jobs:
         run: |
           npm install -g @bytecodealliance/jco
           npm install -g @bytecodealliance/componentize-js
+
       # Build example project(s)
       - name: build project
         run: wash build
         working-directory: examples/${{ matrix.project.lang }}/components/${{ matrix.project.name }}
+
+      # Run the local.wadm that comes with the example
+      - name: test component load
+        if: ${{ matrix.project.test_deploy && matrix.wash-version == 'current' }}
+        shell: bash
+        working-directory: examples/${{ matrix.project.lang }}/components/${{ matrix.project.name }}
+        run: |
+          set -xe
+          wash up &
+          WASH_PID=$!
+          sleep 3;
+          wash app deploy ${{ matrix.project.test_deploy }};
+          TRIES=0
+          while [[ $(wash get inventory --output=json | jq '.inventories[0].components | length') -eq 0 ]] ; do
+            if [[ $TRIES -gt 10 ]]; then
+              echo "❌ failed to find component in inventory output after deploying example manifest";
+              exit -1;
+            fi
+            TRIES=$((TRIES+1));
+            sleep 1;
+          done;
+          echo "✅ successfully started at least one component";
+          wash app delete ${{ matrix.project.test_deploy }};
+          kill $WASH_PID;
```

Comment:
What do you think about including an "invocation command" that you can run here after the app is deployed to test that it actually worked? I don't want to make it too complicated with our examples that require extra dependencies like Redis, but it might be cool to have (at least for the HTTP ones) examples where we `curl` the local port & make sure we get a payload back

Reply:
Hey sorry late getting to this but that would be a great add on -- checking the functionality of the component a little more closely would be great. 

That said, I think if we want to make this more complex we should actually just start making top level tests for these examples, maybe using `cargo xtask` -- we could get rid of the bash spaghetti here and take advantage of the monorepo... What do you think?